test/visibility/private/variables/accessSiblingPrint.good

```diff
@@ -0,0 +1,33 @@
+accessSiblingPrint.chpl:8: Printing symbols visible from here:
+  /Users/diten/chapel3/modules/minimal/internal/ChapelLocale.chpl:27: chpl_sublocID_t
```

Comment:
Heads up, this looks tied to your personal set up rather than something that will work generally

Reply:
Thanks. I'll fix that with a couple more prediffs.